### PR TITLE
Servlet pt.1

### PR DESCRIPTION
**Pull request by clarkandrew10**
Hopefully i'm doing this right ._.

### CODE REVIEW

**Review by jayceb04 — COMMENTED** [review]
Please add a screenshot of the website showing what the changes look like to the PR description.

**Comment by clarkandrew10** [comment]
Let me know if you need anything else

Screen Shot 2021-03-03 at 4 58 56 PM
Screen Shot 2021-03-03 at 4 59 30 PM

**Review by jayceb04 — APPROVED** [review]

**Comment by jayceb04** [comment]
Thanks!

File: portfolio/src/main/java/com/google/sps/servlets/DateServlet.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+@WebServlet("/date")
+public class DateServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+      LocalDateTime myDateObj = LocalDateTime.now();
+      DateTimeFormatter myFormatObj = DateTimeFormatter.ofPattern("dd-MM-yyyy HH:mm:ss");
+      String formattedDate = myDateObj.format(myFormatObj);
+      System.out.println("After formatting: " + formattedDate);
```
Comment (jayceb04) [inline portfolio/src/main/java/com/google/sps/servlets/DateServlet.java:20]: Did you want "After formatting:" in your message?

Comment (clarkandrew10) [reply portfolio/src/main/java/com/google/sps/servlets/DateServlet.java:20]: No, I didn't mean to include that
I had to look into date formatting but I accidentally left that in - It's been adjust in the DateServlet.java file

**Pull request merged**